api/commands/init.ts

```diff
@@ -65,6 +72,23 @@ export async function init(
   fs.writeFileSync(gitignorePath, gitIgnoreContents);
   filesWritten.push(gitignorePath);
 
+  if (options.includeSchedules) {
+    fs.writeFileSync(
+      schedulesJsonPath,
+      prettyJsonStringify(
+        dataform.schedules.Schedules.create({
+          defaultNotification: {
+            emails: [],
```

Comment:
these fields all have the same values as proto3 default values so these lines *shouldn't* be having any effect. (`emails`, `success`, `failure`, and `schedules`)

Reply:
creating an empty json file now